docs/example-narratives/examples/forecast-limits-detailed.json

```diff
@@ -1,34 +1,72 @@
-[
-    {
-        "transmission-facility-id": "line2",
-        "periods": [
+{
+    "snapshot-header": {
+        "last-updated": "2023-07-12T16:00:00-07:00",
+        "snapshot-provider": "X-RC",
```

Comment:
tbh, I didn't question the necessity of this too much, and I am happy to reconsider this addition, since it's essentially the dual of the `ratings-provider` in the proposals that we just added https://github.com/trolie/spec/blob/1.0.0-wip/docs/_data/components/schemas/rating-proposal-header.yaml#L22

In my original yaml vignettes for this exchange, I had given myself the constraint that when you read one of one of these messages, you should be able to tell where it came from, like document-level provenance. We don't have any concrete use cases for routing in the interop regime, so this isn't necessary for that, and I only have a incomplete idea about a validation related use case which is probably better solved with request/response signing. Still, I think document-level provenance is would aid debugging and support.

Building on the rest of your comment, in those vignettes, I had something like a "system of origin" field associated with every `provenance`. https://github.com/misoenergy/TROLIE/blob/main/openapi-spec.yml#L889-L897

I'd like to add that back by making the document-level provenance explicit in the schema. I'll prepare that change as a PR against this PR, then we can decide to remove this entirely or do it properly.